service/history/configs/config.go

```diff
@@ -316,8 +316,8 @@ func NewConfig(dc *dynamicconfig.Collection, numberOfShards int32, isAdvancedVis
 		TimerTaskMaxRetryCount:                            dc.GetIntProperty(dynamicconfig.TimerTaskMaxRetryCount, 100),
 		TimerProcessorEnableSingleCursor:                  dc.GetBoolProperty(dynamicconfig.TimerProcessorEnableSingleCursor, false),
 		TimerProcessorEnablePriorityTaskScheduler:         dc.GetBoolProperty(dynamicconfig.TimerProcessorEnablePriorityTaskScheduler, false),
-		TimerProcessorSchedulerWorkerCount:                dc.GetIntProperty(dynamicconfig.TimerProcessorSchedulerWorkerCount, 200),
-		TimerProcessorSchedulerQueueSize:                  dc.GetIntProperty(dynamicconfig.TimerProcessorSchedulerQueueSize, 10000),
```

Comment:
I originally thought this is the queue size for the weighted channel. But it's the size of the channel that's actually polled by the worker pool. (Task scheduler basically moves tasks from weighted channel to that single channel according to the weight). 

If the number is high, then task scheduler simply move tasks to that single channel in the order they are received. And we lost the effect of wrr scheduler.

I will probably remove this config in the future, I don't think there's a need to tune this number.